File: Java/4-Array-based-Programs/52-smallest-element-in-an-array.java
class Main {
    public static void main(String[] args) {
        int[] arr = {55, 13, 84, 26, 10};
        int small = arr[0];

        for(int i=1; i<arr.length; i++){
            if(arr[i]<small){
                small = arr[i];
            }
        }
        System.out.println("Smallest element is: " + small);
    }
}

// Using built-in method:
// import java.util.Arrays;
// int small = Arrays.stream(arr).min().getAsInt();

File: Python/4-Array-based-Programs/52-smallest-element-in-an-array.py
arr = [20, 85, 97, 15, 60]
small = arr[0]

for i in range(len(arr)):
    if (arr[i]<small):
        small=arr[i]

print("Smallest element is:", small)


# Using built-in method:
# print(min(arr))